pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
```

Comment:
Why `is better` a real loop?

I mean, a real loop `for` plus a nested `if` increase 2 points of cyclomatic complexity.
Using a comprehension list you have just one sentence to assign a variable.